.rhcicd/clowdapp.yaml

```diff
@@ -140,6 +140,40 @@ objects:
                         secretKeyRef:
                           name: policies-ui-backend-db
                           key: ${DB_SECRET_PASSWORD_KEY}
+  - apiVersion: metrics.console.redhat.com/v1alpha1
+    kind: FloorPlan
+    metadata:
+      name: policies-backend-hms
+    spec:
+      database:
+        secretName: ${FLOORIST_DB_SECRET_NAME}
+      objectStore:
+        secretName: ${FLOORIST_HMS_BUCKET_SECRET_NAME}
+      logLevel: ${FLOORIST_LOGLEVEL}
+      suspend: ${{FLOORIST_SUSPEND}}
+      disabled: ${{FLOORIST_DISABLED}}
+      queries:
+        - prefix: hms_analytics/policies/status
+          query: >-
+            SELECT
+              P.id,
+              P.name,
+              P.org_id,
+              P.is_enabled,
+              P.mtime,
+              P.ctime,
+              P.actions,
+              JSON_AGG(PH.host_id) as hosts
```

Comment:
There are records over 1000. I'm wondering whether we should just flatten this:
```
            SELECT
              P.id AS policy_id,
              P.org_id,
              P.is_enabled,
              P.mtime AS policy_mtime,
              P.ctime AS policy_ctime,
              P.actions,
              PH.host_id,
              PH.ctime AS policy_hit_time
            FROM
              policy P
            LEFT JOIN
              policies_history AS PH
            ON
              P.id = uuid(PH.policy_id);
```

This one adds policy hit time and removes policy name.